Scripts/CherwellGetIncident/CherwellGetIncident.py

```diff
@@ -0,0 +1,90 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+args = demisto.args()
+
+# ####################################################################################
+# ############################## CONFIGURATION PART ##################################
+# ####################################################################################
+
+"""
+ `BUSINESS_OBJECT_TYPE` is the name of business object you wish to retrieve using this script.
+ In this case we set it to be 'incident' as this script is in charge of retrieving incidents
+"""
+
+BUSINESS_OBJECT_TYPE = 'Incident'
+
+"""
+ `OUTPUT_FIELDS` should contain all the fields you wish to include in the returned business object.
+ Make sure the field name is identical to the field name in Cherwell system.
+ In order for the fields to appear in the script outputs you will need to to update the script outputs
+ (found in the script settings).
+
+ For example: we wished to retrieve an incident such that the fields: RecordID, Description, Priority,
+ CustomerDisplayName, and so on, will appear in the returned object, thus, we added all of those field names
+ to this `OUTPUT_FIELDS` variable.
+ In addition we added the field names to the script outputs so they will appear as an official output of
+ the script, using the following syntax: Cherwell.BusinessObject.RecordID, Cherwell.BusinessObject.PublicID,
+ Cherwell.BusinessObject.Description, Cherwell.BusinessObject.Priority and so on.
+ Make sure to leave the first part in the output definition (`Cherwell.BusinessObject`) as is.
+"""
+OUTPUT_FIELDS = [
+    'RecordId',
+    'PublicId',
+    'Description',
+    'Priority',
+    'CustomerDisplayName',
+    'OwnedBy',
+    'Service',
+    'CreatedDateTime',
+    'TotalTasks'
+]
+
```

Comment:
I think it is a good idea to put the output path prefix here in a separate var so the user will be abele to change it.
`output_path = "Cherwell.BusinessObjects"`

Reply:
- [x] Check